### PR TITLE
Implement the price component

### PR DESCRIPTION
**Pull request by mikeglopez**
Implements the price component which generates a random price per night as well as rating and number of reviews.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,5 +39,4 @@ From within the root directory:
 ```sh
 npm install -g webpack
 npm install
```
Comment (akcipriano) [inline README.md:41]: Not sure if you want to combine these with the code in Usage

File: client/components/Price.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  margin-bottom: 24px;
+  border-bottom: 1px solid #d2d2d2
+`;
+
+const price = () => faker.random.number({
+  min: 50,
+  max: 1500
+}).toLocaleString();
+
+const Price = () => (
```
Comment (akcipriano) [inline client/components/Price.jsx:31]: You COULD change the name to inform the user stars are being loaded as well? But I understand price is more important so it'd be okay to keep.

**Review by akcipriano — APPROVED** [review]
Everything looks great!

**Review by akcipriano — APPROVED** [review]

File: client/components/Price.jsx (hunk 2 above)
Comment (mikeglopez) [reply client/components/Price.jsx:31]: Thanks for the input! I didn't see these messages until after I merged, but I've implemented the changes and will push them with my next PR! :)

**Pull request merged**